tests/snappi/pfcwd/files/pfcwd_burst_storm_helper.py

```diff
@@ -68,15 +70,21 @@ def run_pfcwd_burst_storm_test(api,
     pause_flow_dur_sec = poll_interval_sec * 0.5
     pause_flow_gap_sec = burst_cycle_sec - pause_flow_dur_sec
 
+    """ Warm up traffic is initially sent before any other traffic to prevent pfcwd 
+    fake alerts caused by non-incremented packet counters during pfcwd detection periods"""
+    warm_up_traffic_dur_sec = WARM_UP_TRAFFIC_DUR
+    warm_up_traffic_delay_sec = 0
+
```

Comment:
Instead of generating "warm up traffic", "data flows" and "PFC storm" in a function, how about using two functions __gen_data_flows and __gen_pause_flow to generate different flows, just like what we did in pfcwd_multi_node_helper.py?

Reply:
I was thinking the same - let me have a look at code re-usability and move around some components